lib/networkCall.js

```diff
@@ -0,0 +1,17 @@
+const fetch = require('node-fetch')
+
+const networkCall = async (url, method = 'get', body = null, headers = null) => {
```

Comment:
What is we need to introduce request timeout here as we interact with external services?

Reply:
Will see how to go about it.